modules/computer/display.c

```diff
@@ -28,18 +28,22 @@ get_glx_info(DisplayInfo *di)
     if (g_spawn_command_line_sync("glxinfo", &output, NULL, NULL, NULL)) {
 	gchar **output_lines;
 	gint i = 0;
+	gint cpd = 0;
 
 	for (output_lines = g_strsplit(output, "\n", 0);
 	     output_lines && output_lines[i];
 	     i++) {
-	    if (strstr(output_lines[i], "OpenGL")) {
+	    if (strstr(output_lines[i], "OpenGL")) 
+	    {
+		if ( strstr(output_lines[i], "profile") ) {    cpd = 1;    }
```

Comment:
I know HardInfo is a mess regarding the coding style, but newer code should be more consistent. Please ensure that:

- There's no space after `(` or before `)`
- `if` statements with a single statement do not get braces
- `if` statements should be on a line by themselves

Reply:
fixed, i must point that the code was made before the source styling fixation..that's why ...